### DISCUSSION TITLE
annoy

### DISCUSSION THREAD

**Discussion by wanghongqi8548488** [Chia-Network/chia-blockchain#4458] [Support]
I didn't back up Chia's mnemonics and key public keys. Only P good files, are these files useless

**Comment by Hadamcik** [comment]
If you don't have any access to private keys, you have no way of using your existing plots or accessing your coins.

**Comment by wanghongqi8548488** [comment]
> 如果你没有任何私人钥匙，你就没有办法使用你现有的阴谋或获得你的硬币。

That's basically invalid?

**Comment by wanghongqi8548488** [comment]
5t

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.

**Comment by wanghongqi8548488** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
